apps/demo-game/src/pages/admin/reports/[id].tsx

```
@@ -0,0 +1,458 @@
+  const segmentResultsPerPlayer = game.players.map((player) => {
+    return segmentEndResults.specificResults
+      .filter((result) => result.player.id === player.id)
+      .map((result) => {
+        return Object.values(result.facts.decisions).map((decision, _, arr) => {
+          return Number(decision) / arr.length
+        })
+      })
+  })
+  const segmentResultPerPlayerAvg = segmentResultsPerPlayer.map((arr) => {
+    return arr.reduce((acc, val, ix) => {
+      val.map((v, i) => (acc[i] += v / arr.length))
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)